<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtyRestock</class>
 <widget class="QWidget" name="QtyRestock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>95</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>95</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background: #fff
}

QLabel {
	color: #fff;
	background: rgb(0, 159, 161);
	padding-left: 5px;
	padding-right: 5px;
}

QLineEdit {
	border: 1px solid rgb(0, 159, 161);
	border-radius: 10px;
	padding-left: 10px;
	color: rgb(0, 159, 161)
}

QLineEdit:hover {
	border: 2px solid rgb(0, 159, 161);
}

QPushButton {
	border: 1px solid;
	margin: 0;
	height: 40px;
	padding-left: 5px;
	padding-right: 5px;
	border-radius: 5px;
}

#add_stock_button {
	color: rgb(0, 159, 161);
	border-color: rgb(0, 159, 161);
}

#add_stock_button:hover{
	background: rgb(0, 159, 161);
	color: #fff;
}

#remove_stock_button {
	border-color: rgb(224, 27, 36);
	color: rgb(224, 27, 36);
}

#remove_stock_button:hover {
	color:#fff;
	background-color: rgb(224, 27, 36);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Enter quantity taken from or added to the inventory</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="qty_label">
       <property name="maximumSize">
        <size>
         <width>356356</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Qty</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="qty_input">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="add_stock_button">
       <property name="maximumSize">
        <size>
         <width>134</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Add Stock</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove_stock_button">
       <property name="maximumSize">
        <size>
         <width>161</width>
         <height>45</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Remove Stock</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
